docs/ASSEMBLY.md

```diff
@@ -15,46 +15,62 @@ Below are instructions for assembling your Sprig console.
 | 1        | Raspberry Pi Pico                                                         |
 | 4        | clear backings (2 with D-pad cutouts, 2 solid)                            |
 | 1        | bag of 10 tactile button switches                                         |
-| 1        | bag of hardware for screen (2 spacers, 2 M2\*14mm screws, 2 hex nuts)\*    |
-| 1        | bag of hardware for backings (8 M2\*10mm screws, 8 hex nuts, allen key)\* |
+| 1        | bag of hardware for screen (2 spacers, 2 M2\*14mm bolts, 2 hex nuts)\*    |
+| 1        | bag of hardware for backings (8 M2\*10mm bolts, 8 hex nuts, allen key)\* |
 | 1        | adapter or micro-USB cable                                                |
 | 1        | bag of 2 AAA batteries                                                    |
 
-\* May include extra hardware.
+> [!NOTE]
+> Extra hardware may be included - it's all packed by hand. If you're missing a part, [DM `@graham` on the Hack Club Slack](https://hackclub.slack.com/team/U04QH1TTMBP) or email graham<img src="https://upload.wikimedia.org/wikipedia/commons/8/88/At_sign.svg" height=15px alt="@"></img>hackclub.com. 
```

Comment:
The weird trick here with the at sign being an image is so crawlers don't see the email address as one. The image's alt text is `@`, so it's totally transparent when you copy and paste it.

Reply:
please modify this copy to point ppl to the #sprig channel instead of yours truly